src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -1908,6 +1924,7 @@ void ov::npuw::LLMCompiledModel::serialize(std::ostream& stream, const ov::npuw:
         write(model_stream, m_prefix_caching_max_num_blocks);
         write(model_stream, m_gemma_sliding_window_size);
         write(model_stream, m_is_whisper);
+        write(model_stream, m_model_rt_info);
```

Comment:
I don't think this is a good idea.
rt info could contain a lot of information which is not directly affects compilation or even have no sense for deserialized model.